go/batch-submitter/drivers/max_priority_fee_fallback.go

```diff
@@ -0,0 +1,26 @@
+package drivers
+
+import (
+	"errors"
+	"math/big"
+	"strings"
+)
+
+var (
+	errMaxPriorityFeePerGasNotFound = errors.New(
+		"Method eth_maxPriorityFeePerGas not found",
+	)
+
+	// FallbackGasTipCap is the default fallback gasTipCap used when we are
+	// unable to query an L1 backend for a suggested gasTipCap.
+	FallbackGasTipCap = big.NewInt(1500000000)
```

Comment:
@tynes is the best we can do in terms of a fallback? i.e. using a constant, or is there a better way you can think of?